### PR TITLE
Setting up gonode CLI tool

### PR DESCRIPTION
**Pull request by edentech88**
## Purpose:
**A. Create cross platform command line application (go) that accept parameters in the groups**

Preliminary set of command groups:

install
update
init
start
stop
show
interactive
Commands for each group will be defined in the follow up tasks

**B. All outputs should be user friendly formatted and colored (research and integrate library for good output)**

### CODE REVIEW

File: cmd/init.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		}
+
+		if err := log.SetLevelName(config.LogLevel); err != nil {
+			return errors.Errorf("--log-level %q, %v", config.LogLevel, err)
+		}
+
+		log.Info("flag-name: ", workDirectoryFlag)
+
+		return runInit(ctx, config)
+	})
```
Comment (levkoburburas) [inline cmd/init.go:64]: Why do you repeat this code in every command? These are global flags.

pastel-utility [global args] [command] [command args]

File: cmd/init.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+	"github.com/pastelnetwork/pastel-utility/configs"
+	"github.com/pastelnetwork/gonode/common/cli"
+	"github.com/pastelnetwork/gonode/common/log"
+	"github.com/pastelnetwork/gonode/common/log/hooks"
+	"github.com/pastelnetwork/gonode/common/sys"
+	"github.com/pkg/errors"
+)
+
+func setupInitCommand(app *cli.App, config *configs.Config) {
```
Comment (levkoburburas) [inline cmd/init.go:15]:
```suggestion
func setupInitCommand(config *configs.Config) *cli.Command {
```

File: cmd/stop.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+		log.Info("flag-name: ", stopFlag)
+
+		return runStop(ctx, config)
+	})
+	app.AddCommands(stopCommand)
+}
+
+func runStop(ctx context.Context, config *configs.Config) error {
+	log.WithContext(ctx).Info("Stop")
+	defer log.WithContext(ctx).Info("End")
```
Comment (levkoburburas) [inline cmd/stop.go:55]: I'm afraid you didn't understand for what it and how it works

**Review by levkoburburas — CHANGES_REQUESTED** [review]
This mess. Almost all of the code needs to be rewritten.

**Comment by edentech88** [comment]
@levkoburburas be patient now. you or I are

**Comment by edentech88** [comment]
@levkoburburas sorry my bad, not everything is done in this PR. The PR https://github.com/pastelnetwork/pastel-utility/pull/4 is waiting for your review (Consider that's not ready to merge)

**Pull request closed without merging**